.travis.yml

```diff
@@ -1,5 +1,4 @@
 sudo: false
-cache: bundler
```

Comment:
The reason that it is failing is that you changed the gemspec but didn't committed the changes in the Gemfile.lock. Could you add that changes?